airflow-core/src/airflow/executors/local_executor.py

```diff
@@ -108,6 +108,10 @@ def _execute_work(log: logging.Logger, workload: workloads.ExecuteTask) -> None:
     from airflow.sdk.execution_time.supervisor import supervise
 
     setproctitle(f"airflow worker -- LocalExecutor: {workload.ti.id}")
+
+    base_url = conf.get("api", "base_url", fallback="/")
+    default_execution_api_server = f"{base_url.rstrip('/')}/execution/"
```

Comment:
Can we just add a property or extract this onto the base executor or somewhere relevant?
Too many repetitions. It will be easy to miss in case of a refactor.

Reply:
The problem is that they are with separate providers. So keeping the logic in central location has a big disadvantage where that provider won't work until that version is released!

So the repetition is intentional

Reply:
Valid point, seems fine. Yeah we will not be able to do that.